.travis.yml

```diff
@@ -68,11 +63,11 @@ cache:
     - $HOME/.composer/cache
 
 before_install:
-  - echo "extension = redis.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-  - phpenv config-rm xdebug.ini
+    - echo "extension = redis.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
```

Comment:
Remove the travis.yml.
* Enable Github Action use as a source https://github.com/spryker-eco/avalara-tax/blob/master/.github/workflows/ci.yml.
* https://github.com/spryker-eco/avalara-tax/blob/master/composer.json